cni-plugin/pkg/dataplane/linux/dataplane_linux.go

```diff
@@ -222,6 +232,17 @@ func (d *linuxDataplane) DoNetworking(
 				return fmt.Errorf("failed to set net.ipv6.conf.lo.disable_ipv6=0: %s", err)
 			}
 
+			err = hostNS.Do(func(_ ns.NetNS) error {
+				if err := writeProcSys(fmt.Sprintf("/proc/sys/net/ipv6/conf/%s/disable_ipv6", hostVethName), "0"); err != nil {
```

Comment:
Is this an independent fix, or a necessary part of the main change here?

Reply:
It is not a fix, it is part of the improvement. Since the host veth now is created in the host namespace, we need to switch to it to enable ipv6 on host veth. Before this change, since both veths were in the container namespace, the line below was sufficient for both interfaces: https://github.com/projectcalico/calico/blob/03b78052790cb845d186bd1103e7fe15ce55c665/cni-plugin/pkg/dataplane/linux/dataplane_linux.go#L213